src/assets/styles/themes/lup-rebrand.scss

```diff
@@ -0,0 +1,125 @@
+/** Rebranding work done for the Land Use Planning platform as outlined in DESENG-53. **/
+
+// Main Colors.
```

Comment:
Black is given a variable so many times here. I suppose this is necessary because we're doing a retrofit, and if we went back then we would want different values. If that's not the case, perhaps we could consolidate some of the black colour variables in a future ticket.

Reply:
Great point here. If we have the cycles for a scss clean-up, we could definitely do a lot to reduce code debt